app/controllers/stages_controller.rb

```diff
@@ -133,9 +129,8 @@ def stage_params
       :update_github_pull_requests,
       :comment_on_zendesk_tickets,
       command_ids: [],
-      flowdock_flows_attributes: [:id, :name, :token, :_destroy],
-      new_relic_applications_attributes: [:id, :name, :_destroy]
-    )
+      new_relic_applications_attributes: [:id, :name, :_destroy],
```

Comment:
I saw your justification for this on your earlier PR, but it does break with the ruby style guide: https://github.com/bbatsov/ruby-style-guide#no-trailing-params-comma